e2e/mock-api/src/app/responses.js

```diff
@@ -63,6 +63,52 @@ export const createStepUpUrl = (url) => {
   return redirectUrl.toString();
 };
 
+export const createTreeStepUpUrl = (url) => {
```

Comment:
As part of the test fixes for transactional / tree-based authorization I decided to split a few things out based on whether they are creating headers or URLs for tree-based or transactional authorization use cases.